_includes/manuals/1.13/4.1.2_roles_and_permissions.md

```diff
@@ -12,6 +12,8 @@ These may be created, deleted and edited on the **Roles** page. Each role will c
 
 There is one built-in system role, 'Default role'. This is a set of permissions that every user will be granted, in addition to any other roles that they have.
 
+Roles can be also associated to Locations or Organizations if these are allowed. Unlike other objects this does not mean that Roles would be only available in a particular scope. Roles are always global for the whole Foreman. The association means that filters of such role are scoped to a particular Organization or Location. Imagine you want to create a role representing Manager of Organization A. You can clone existing Manager role and associate it with Organization A. If you later assign this role to some users, they will be granted manager permissions but only on resources of Organization A. Note that some resource are not scopeable by Organization and Locations. Filters for such resources grants permissions globally.
```

Comment:
typos:

'You can clone *an* existing Manager role'
 'filters for such resources *grant* permissions globally'  
"note that some *resourceS* are not scopeable".